src/components/dashboard/SendLinkSummary.js

```diff
@@ -40,7 +40,8 @@ const SendLinkSummary = ({ screenProps, styles }: AmountProps) => {
   const [link, setLink] = useState('')
 
   const { goToRoot, navigateTo } = screenProps
-  const { fullName } = useProfile()
+  const { fullName, email } = useProfile()
```

Comment:
email and full name are supposed to come from the input form.
when the user opens a vendor payment link, he can fill the name + email he wants to send to vendor

Reply:
@TomasGonzalez 1. add such form fields to the form is showing when user opens vernod's link 2. prefill them from profile via useProfile 3. on form submit pass their values to the SendLinkSummary through the screenState 4. add  useCachedScreenState to SendLinkSummary to prevent values loss if page refreshed (see src/components/dashboard/TransactionConfirmation.js as an example)